### DISCUSSION TITLE
How do you keep a floating panel half screen and prevent it from being resized with a gesture?

### DISCUSSION THREAD

**Discussion by AustinConlon** [scenee/FloatingPanel#485] [Q&A] [answered]
I have a dismiss button and would like to rely on that.

**Comment by scenee** [comment, marked as answer]
Thank you for your question. I could found a bug not to keep the pan gesture disabled on this use case. I created #486  to fix it.

**Reply by AustinConlon** [reply]
As a workaround, I could set the `gestureRecognizers` property of the floating panel controller's `surfaceView` to `nil`.
